### PR TITLE
fix: disappearing line break made with shortcut Shift-Enter

### PR DESCRIPTION
**Pull request by bojangles-m**
https://app.clickup.com/t/2523021/TASK-7703

### CODE REVIEW

File: .changeset/chilly-bananas-mix.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@frontify/fondue': patch
+---
+
+fixed Richtext editor: disappearing line break made with shortcut Shift-Enter
```
Comment (ragi96) [inline .changeset/chilly-bananas-mix.md:5]:
```suggestion
fix(RichTextEditor): disappearing line break made with shortcut Shift-Enter
```

File: packages/fondue/src/components/RichTextEditor/serializer/serializeToHtml.spec.ts [hunk 2]
```diff
@@ -49,17 +49,41 @@ describe('serializeNodesToHtml()', () => {
...
     });
 
-    it('serializes an empty p node to a paragraph with an zero width space', () => {
-        const node = [
-            {
-                type: 'p',
-                children: [{ text: '' }],
-            },
-        ];
+    describe('paragraf', () => {
```
Comment (ragi96) [inline packages/fondue/src/components/RichTextEditor/serializer/serializeToHtml.spec.ts:52]:
```suggestion
    describe('paragraph, () => {
```

File: packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts [hunk 3]
```diff
@@ -61,8 +61,14 @@ describe('Markdown to slate Transformer', () => {
     });
 
     it('should transform paragraph', () => {
```
Comment (ragi96) [inline packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts:63]:
```typescript
 const testCases = [
        { markdown: codeBlockMarkdown[0], expectedTree: codeBlockTree[0] },
        { markdown: codeBlockMarkdown[1], expectedTree: codeBlockTree[1] },
        { markdown: codeBlockMarkdown[2], expectedTree: codeBlockTree[2] },
    ];
    
    testCases.forEach(({ markdown, expectedTree }) => {
        it(`should transform code block`, () => {
            const result = transformer.process(markdown);
            expect(result).to.deep.equal(expectedTree);
        });
    });
``` 
How about that?

Would do it like that also for the other tests that are adjusted, as its odd to have multiple expects, or if there are different things you test then split it to multiple tests with proper naming

Comment (bojangles-m) [reply packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts:63]: I think you can not do that in cypress, just vi

File: packages/fondue/src/components/RichTextEditor/serializer/serializeToHtml.spec.ts (hunk 2 above)
Comment (bojangles-m) [reply packages/fondue/src/components/RichTextEditor/serializer/serializeToHtml.spec.ts]: Updated

File: .changeset/chilly-bananas-mix.md (hunk 1 above)
Comment (bojangles-m) [reply .changeset/chilly-bananas-mix.md]: Thnx, updated

File: packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts (hunk 3 above)
Comment (bojangles-m) [reply packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts]: Updated

**Review by ragi96 — APPROVED** [review]
LGTM

File: packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts (hunk 3 above)
Comment (ragi96) [reply packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts]: Nice, thanks 👍

**Review by noahwaldner — APPROVED** [review]

**Pull request merged**